packages/ng/applications/sandbox/src/app/issues/date-picker-width/date-picker-width.component.ts

```diff
@@ -5,6 +5,7 @@ import { Component } from '@angular/core';
 	templateUrl: './date-picker-width.component.html'
 })
 export class DatePickerWidthComponent {
+	true = true;
```

Comment:
wut ?

Reply:
not really needed, its in the sandbox so idc and dont feel like i need to fix it